confd/warp/warp.go

```diff
@@ -0,0 +1,228 @@
+package warp
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"strings"
+
+	. "github.com/cloudogu/ces-confd/confd"
+	"github.com/coreos/etcd/client"
+	"github.com/pkg/errors"
+	"golang.org/x/net/context"
+)
+
+type Configuration struct {
```

Comment:
**[checkstyle]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type Configuration should have comment or be unexported